### ISSUE TITLE
neovim: wrapped configuration's NVIM_SYSTEM_RPLUGIN_MANIFEST/rplugin.vim does not exist

### ISSUE THREAD

**Issue by berbiche** [nix-community/home-manager#1789]
### Issue description

My neovim configuration is pretty mundane, but when I launch it I get this error:

```
Error detected while processing function <SNR>17_LoadRemotePlugins:
line    3:
E484: Can't open file /nix/store/r8iw2m8dy18r4gi7xpzxx21j2l243b6m-neovim-0.4.4/rplugin.vim
```

https://github.com/NixOS/nixpkgs/blob/8c8731330b53ba0061686f36f10f101e662a4717/pkgs/applications/editors/neovim/wrapper.nix#L37

It seems as though `$out/rplugin.vim` failed to be generated.

Configuration:

``` nix
{
  programs.neovim = {
    enable = true;
    viAlias = true;
    vimAlias = true;
    vimdiffAlias = true;
    withNodeJs = true;

    plugins = [ "<snip>" ];

    extraConfig = ''<snip>'';
  };
}
```

### Meta

I wasn't sure whether to open this issue against HM or nixpkgs.

I've had this issue for ~2 months now but always brushed it off and pressed `<CR>` to continue.

#### Maintainer CC

@teto

**Comment by teto** [comment]
the file should be generaterd in every case so that's weird. I dont know why

**Comment by berbiche** [comment]
``` console
$ find /nix/store/path-to-neovim-0.4.4/ -name '*rplugin*'
/nix/store/r8iw2m8dy18r4gi7xpzxx21j2l243b6m-neovim-0.4.4/share/nvim/runtime/plugin/rplugin.vim
```

So the file exists, but if I understand the logic in the package, this isn't the one that's being generated?

**Comment by teto** [comment]
that file is part of the neovim runtime, that's not the one you are looking for. For instance mine is in /nix/store/ajkjs0qwiz8aklfqvmg2ia5arg0yrmcd-neovim-master/rplugin.vim . 
You may want to track down the value of manifestRc since it is in fact generated depending on manifestRc. I think it should be != null but looks like in your case manifestRc is null ?!

**Comment by azahi** [comment]
I only get this error if I have `nixpkgs.vimPlugins.vim-wakatime` present in the plugins section. @berbiche could this be the case for you as well? I guess this should be a nixpkgs bug.

**Comment by berbiche** [comment]
Yup, looks like `vim-wakatime` is the culprit :+1:.

I'll move this issue to nixpkgs.

**Comment by berbiche** [comment]
This doesn't seem to be an issue with `vim-wakatime` but with the wrapper generating the configuration in nixpkgs.

Looking at the source code of https://github.com/NixOS/nixpkgs/blob/1299c42d88a220f887db3ca435dd6cbafdbca323/pkgs/applications/editors/neovim/wrapper.nix#L63-L100, the rplugin manifest generation can indeed fail.

**Comment by berbiche** [comment]
Fixed by nixos/nixpkgs#124990
